pom.xml

```diff
@@ -105,12 +106,17 @@
 			<artifactId>spring-boot-starter-hateoas</artifactId>
 			<scope>compile</scope>
 		</dependency>
-
 		<dependency>
 			<groupId>uk.gov.companieshouse</groupId>
 			<artifactId>company-accounts-library</artifactId>
 			<version>${company-accounts-library.version}</version>
 		</dependency>
+		<dependency>
+			<groupId>uk.gov.companieshouse</groupId>
+			<artifactId>document-generator-core</artifactId>
```

Comment:
Which version of the document-generator-core is this dependent on? The old library is being replaced by a submodule in [document-generator](https://github.com/companieshouse/document-generator). It won't function as a library anymore as the architecture is changing to a plug-in based approach.

Reply:
i will remove the dependency.

Reply:
adf92c4